staging/src/k8s.io/code-generator/cmd/register-gen/generators/targets.go

```diff
@@ -115,7 +115,7 @@ func GetTargets(context *generator.Context, args *args.Args) []generator.Target
 							gv:              gv,
 							typesToGenerate: typesToRegister,
 							outputPackage:   pkg.Path,
-							imports:         generator.NewImportTracker(),
+							imports:         generator.NewImportTrackerForPackage(pkg.Path),
```

Comment:
xref breakage in register-gen associated with this gengo bump... https://github.com/kubernetes/kubernetes/issues/129290

unnoticed because nothing in-tree apparently uses register-gen